rabbitmq/hatch.toml

```diff
@@ -7,11 +7,17 @@ dependencies = [
 
 [[envs.default.matrix]]
 python = ["2.7", "3.8"]
-version = ["3.5", "3.6", "3.7"]
+version = ["3.5", "3.6", "3.7", "3.8", "3.9", "3.10", "3.11"]
+
+[[envs.default.matrix]]
```

Comment:
let me know if you think this needs an explanatory comment

Reply:
I do, a comment will probably be helpful in the future and save us some time 🙂